pkg/kf/commands/apps/push.go

```diff
@@ -127,11 +129,28 @@ func NewPushCommand(p *config.KfParams, pusher kf.Pusher, b SrcImageBuilder) *co
 					}
 				}
 
+				var envMap map[string]string
```

Comment:
consider deferring defining this until you need it on 138

Reply:
done